unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt

```diff
@@ -886,7 +886,7 @@ $punct  ⊇ [[\u0021-\u007E] - [0-9 A-Z a-z]]
 
 # Ideographic invariants
 
-Let $ideohack = [〆 〇 〡-〩]
+Let $ideohack = [〆 〇 〡-〩 \p{Name=/^CHINESE SMALL (SIMPLIFIED|TRADITIONAL) ER$/}]
```

Comment:
add a comment for this addition under line 893? or remove the existing change log there??

Reply:
Removed it, it really doesn’t give any information that isn’t in the UnicodeSet expression except for figuring out when that invariant was added (terminus post quem: Tangut, 9.0; terminus ante quem: Nushu, 10.0).